pkg/gcv/configs/config.go

```diff
@@ -204,9 +225,31 @@ func arrayFilterSuffix(arr []string, suffix string) []string {
 	return filteredList
 }
 
-// listFiles returns a list of files under a dir. Errors will be grpc errors.
+// listFiles returns a list of files under a dir either locally or in GCS. Errors will be grpc errors.
 func listFiles(dir string) ([]string, error) {
 	var files []string
+	var err error
+
+	if strings.HasPrefix(dir, "gs://") {
```

Comment:
Please use https://golang.org/pkg/net/url/#Parse for determining if the scheme is "gs".